bitcoin/src/blockdata/opcodes.rs

```diff
@@ -16,11 +16,9 @@
 use core::{fmt, convert::From};
 use bitcoin_internals::debug_from_display;
 
-// Note: I am deliberately not implementing PartialOrd or Ord on the
-//       opcode enum. If you want to check ranges of opcodes, etc.,
-//       write an #[inline] helper function which casts to u8s.
-
 /// A script Opcode.
+// Note: I am deliberately not implementing PartialOrd or Ord. If you want to check ranges of
+//       opcodes, etc., write an #[inline] helper function which casts to u8s.
```

Comment:
I wonder what's the reason though.

Reply:
No idea, @apoelstra did you write this originally?

Reply:
The reason for not having `Ord` is that there is no "ordering" of opcodes that makes conceptual sense, and I can't think of a reason to put sets of individual opcodes into ordered data structures (or sorting them directly).

The reason I **mentioned** this is because Bitcoin Core routinely uses `<` and `>` on opcodes to effect consensus logic, something which hides implicit assumptions about the opcodes' byte encoding, in code where the exact behavior is critical, and dresses this up in a skinsuit of mathematical notation.

Probably nobody else is aware of, or as offended by, this particular quirk of Core as I was, so we can drop the comment :). Also like, I'm pretty sure we have the same logic, I just had to add some `as u8`s before the `<`s.

Reply:
> Also like, I'm pretty sure we have the same logic, I just had to add some `as u8`s before the `<`s

Made me laugh :)